ios/VolumeManager.m

```diff
@@ -26,6 +26,7 @@ - (instancetype)init {
 
 - (void)initVolumeView {
   volumeView = [[MPVolumeView alloc] init];
+  volumeView.transform = CGAffineTransformMakeScale(0.1, 0.1);
```

Comment:
What happens if you pass 0.0 instead?

Reply:
When I passed 0.0 for some reason the iOS Airplay popup appears when you click Show Native UI 🤔 

![IMG_1742](https://user-images.githubusercontent.com/61282033/229998004-e749e4fe-b430-4f22-93b8-de9aaacc1ca9.jpg)

Reply:
@hirbod Apologies, I forgot to tag you in my reply above